frontend/packages/pipelines-plugin/src/components/pipelines/detail-page-tabs/EventListenerDetails.tsx

```diff
@@ -11,6 +11,7 @@ export interface EventListenerDetailsProps {
 
 const EventListenerDetails: React.FC<EventListenerDetailsProps> = ({ obj: eventListener }) => {
   const { t } = useTranslation();
+  const triggers = eventListener.spec?.triggers?.filter((trigger) => trigger?.template?.name) || [];
```

Comment:
Nit: `spec` isn't optional.
```suggestion
  const triggers = eventListener.spec.triggers?.filter((trigger) => trigger?.template?.name) || [];
```